server/src/main/scala/services/exports/flights/ArrivalToCsv.scala

```diff
@@ -9,7 +10,7 @@ object ArrivalToCsv {
   val rawArrivalHeadingsWithTransfer: String = rawArrivalHeadings + ",Transfer Pax"
 
   def arrivalAsRawCsvValues(arrival: Arrival, millisToDateOnly: MillisSinceEpoch => String,
```

Comment:
I presume this function is still used for the arrival feed exports or something?

Reply:
correct.